deploy/roles/python36-scl/meta/main.yml

```diff
@@ -0,0 +1,60 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
```

Comment:
@dvbalen Might as well fill this out since we've included the galaxy metadata?

Reply:
CC0 License?

Reply:
Yep, that's our standard.